### PR TITLE
User preview in popup

### LINKED ISSUE #91: User preview in popup

### PR DESCRIPTION
**Pull request by Katrin-kudryash**
Resolve #91

### CODE REVIEW

**Review by awinogradov — APPROVED** [review]

File: src/components/users/UserTeams.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                            {team.roles.map((role) => (
+                                <Text size="s" key={role.title} color={gray10}>
+                                    {role.title},
+                                </Text>
+                            ))}
+                        </StyledRoles>
+                    </div>
+                ))}
+
+                <InlineTrigger icon={<PlusIcon noWrap size="xs" />} text={'Add teams'} onClick={() => {}} />
```
Comment (9teen90nine) [inline src/components/users/UserTeams.tsx:77]: todo comment

File: src/pages/api/user/group/[groupId].ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
```
Comment (9teen90nine) [inline src/pages/api/user/group/[groupId].ts:1]: no more API pages, use trpc backend

**Review by 9teen90nine — CHANGES_REQUESTED** [review]
No more api pages!

File: src/pages/api/user/group/[groupId].ts (hunk 2 above)
Comment (Katrin-kudryash) [reply src/pages/api/user/group/[groupId].ts]: ok, done

**Review by 9teen90nine — APPROVED** [review]

**Pull request merged**